phpBB/phpbb/search/fulltext_mysql.php

```diff
@@ -78,12 +84,14 @@ class fulltext_mysql extends \phpbb\search\base
 	 * @param \phpbb\auth\auth $auth Auth object
 	 * @param \phpbb\config\config $config Config object
 	 * @param \phpbb\db\driver\driver_interface Database object
+	 * @param \phpbb\event\dispatcher_interface	$phpbb_dispatcher	Event dispatcher object
 	 * @param \phpbb\user $user User object
 	 */
-	public function __construct(&$error, $phpbb_root_path, $phpEx, $auth, $config, $db, $user)
+	public function __construct(&$error, $phpbb_root_path, $phpEx, $auth, $config, $db, $phpbb_dispatcher, $user)
```

Comment:
The best solution is to move everything here to services with a factory but it's out of the scope of  this PR (and can't be done in 3.1 anyway).

About the argument, add it to the end please. Calling a function with too many arguments isn't an issue.